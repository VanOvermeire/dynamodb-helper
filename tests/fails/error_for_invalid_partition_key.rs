use dynamodb_helper::DynamoDb;

#[derive(DynamoDb)]
pub struct Example {
    #[partition]
    invalid: Vec<String>,
    first: String,
    second: u32,
}

fn main() {}
