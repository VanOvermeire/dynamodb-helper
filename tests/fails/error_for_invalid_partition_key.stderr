error: DynamoDB only supports strings, numbers and booleans as keys
 --> tests/fails/error_for_invalid_partition_key.rs:6:5
  |
6 |     invalid: Vec<String>,
  |     ^^^^^^^
